packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -146,6 +147,14 @@ Use `size="large"` Form Pills only for specific and approved use cases, such as
   {largeExample}
 </LivePreview>
 
+### Tree Variant 
```

Comment:
suggestion: Move this section between "Selectable and dismissible" and "Internationalization"